backend/src/routes/api/notebooks/index.ts

```diff
@@ -51,20 +51,22 @@ module.exports = async (fastify: KubeFastifyInstance) => {
           ? false
           : await getNotebookStatus(fastify, namespace, name);
 
-        const notebookName = notebook.metadata.name;
-        if (isRunning && !notebook.metadata.annotations['opendatahub.io/link']) {
-          getRoute(fastify, namespace, notebookName)
-            .then((route) => {
-              patchNotebookRoute(fastify, route, namespace, notebookName).catch((e) => {
-                fastify.log.warn(`Failed patching route to notebook ${notebookName}, ${e.message}`);
-              });
-            })
-            .catch((e) => {
-              fastify.log.warn(`Failed getting route ${notebookName}, ${e.message}`);
-            });
+        const notebookName = notebook?.metadata.name;
+        let newNotebook: Notebook | void;
```

Comment:
Nit, `void` is for functions that return nothing... `undefined` is for variables without a value.